app/controllers/api/v1/orders_controller.rb

```diff
@@ -153,6 +153,15 @@ def serializer
         Api::V1::OrderSerializer
       end
 
+      def shallow_serializer
+        Api::V1::OrderShallowSerializer
+      end
+
+      def select_serializer
+        is_shallow_render = params[:shallow] == 'true'
```

Comment:
How about using `params[:shallow] == 'true' ? shallow_serializer : serializer` ? 🤔 💭

Reply:
Personal preference. Creating the variable improves readability.

Reply:
I feel its unnecessary variable assignment.